### PR TITLE
REV-551 fix garages with zero value

### PR DESCRIPTION
**Pull request by FarahJamal-business**
### Jira URL

https://optimumpartners.atlassian.net/browse/REV-551
### Description

garages with zero value didn't appear

### CODE REVIEW

File: src/components/agent-details/CardInformation/index.jsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ function CardInformation({
               <BathtubIcon />
             </IconGroup>
           )}
-          {garages ? (
+          {garages >= 0 ? (
```
Comment (mohdabed123) [inline src/components/agent-details/CardInformation/index.jsx:79]:
```suggestion
          {!!garages ? (
```

File: src/components/home/FeaturedListingCard/index.jsx [hunk 2]
```diff
@@ -99,7 +99,7 @@ function Card({ data }) {
                   <BathtubIcon />
                 </FeaturedIconContainer>
               )}
-              {garages && (
+              {garages >= 0 && (
```
Comment (mohdabed123) [inline src/components/home/FeaturedListingCard/index.jsx:102]:
```suggestion
              {!!garages &&  (
```

File: src/components/home/FeaturedListingCard/index.jsx [hunk 3]
```diff
@@ -144,11 +144,13 @@ function Card({ data }) {
                   <BathtubIcon />
                 </FeaturedIconContainer>
               )}
-              {garages && (
+
+              {garages >= 0 && (
```
Comment (mohdabed123) [inline src/components/home/FeaturedListingCard/index.jsx:148]:
```suggestion
              {!!garages  && (
```

File: src/components/listing-search-page/Card/index.jsx [hunk 4]
```diff
@@ -85,7 +85,7 @@ function Card({ data }) {
               <Bathtub />
             </IconsNumber>
           )}
-          {garages && (
+          {garages >= 0 && (
```
Comment (mohdabed123) [inline src/components/listing-search-page/Card/index.jsx:88]:
```suggestion
          {!!garages  && (
```

File: src/components/listing-search-page/MapCard/index.jsx [hunk 5]
```diff
@@ -60,7 +60,7 @@ function MapCard({ data }) {
           </IconsNumber>
         )}
 
-        {garages && (
+        {garages >= 0 && (
```
Comment (mohdabed123) [inline src/components/listing-search-page/MapCard/index.jsx:63]:
```suggestion
        {!!garages && (
```

**Review by mohdabed123 — CHANGES_REQUESTED** [review]
0 is falsy so using double !! should do the job

File: src/components/agent-details/CardInformation/index.jsx (hunk 1 above)
Comment (FarahJamal-business) [reply src/components/agent-details/CardInformation/index.jsx:79]: that's remove garage icon and eslint rules didn't accept it

**Review by mohdabed123 — APPROVED** [review]

**Pull request merged**